message PixelClockInfoBuffer {
  optional int32 channel_id = 1;
  optional int64 time_stamp = 2;
  
  required int32 direction = 3;  
}